src/apps/vpn/mozillavpn.cpp

```diff
@@ -941,9 +942,12 @@ void MozillaVPN::postAuthenticationCompleted() {
 void MozillaVPN::mainWindowLoaded() {
   logger.debug() << "main window loaded";
 
+  Telemetry::stopTimeToMainScreenTimer();
```

Comment:
Main is probably misleading. Because it sounds like the home screen with the ring and the toggle button. Maybe we want to call it: stopTimeToFirstScreenTimer